.gitignore

```diff
@@ -1,4 +1,3 @@
-Dockerfile.*
```

Comment:
[test.sh](https://github.com/cyberark/conjur-cli/blob/v4/test.sh#L14) creates Dockerfiles that this tells git to ignore. Instead of deleting this line, can you change it to match the names of the created files, please?